docs/infra/discourse.md

```diff
@@ -42,6 +42,13 @@ We do not use config files to maintain Discourse.
 The config settings are dynamic and stored inside the image container.
 If a rollback is needed, see [Backups](#backups).
 
+### Firewall
+
+Note that DigitalOcean maintains its own firewall, a layer above the Linux Droplet where the Discourse forum is hosted.
+If adding or changing services on the Droplet host, make sure the right ports are open on the host from the DigitalOcean admin panel.
```

Comment:
Unless the ports will be given on the admin panel, Is it possible to comment on the "right ports"?

Reply:
@10eMyrT It was a little confusing. I rephrased it in commit d730f227c0a788eb45308532fde13571794f3d3d!